recipes/quast/meta.yaml

```diff
@@ -1,44 +1,51 @@
+{% set name = "quast" %}
+{% set version = "4.5" %}
+{% set hash = "873ed8cd69e2d92fb87d75a37ecb75ea" %}
```

Comment:
The hash also changes at a new release, so I guess it's better to keep it there.
The name is indeed not necessary, but it also doesn't conflict right now, so I'll leave it for now.